Sources/BraintreeVenmo/BTVenmoLineItem.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  BTVenmoLineItem.swift
+//  BraintreeVenmo
+//
+//  Created by Khushboo Mukesh Chandwani on 3/23/23.
+//
+
+import Foundation
+
+/// Use this option to specify whether a line item is a debit (sale) or credit (refund) to the customer.
+@objc public enum BTVenmoLineItemKind: Int {
+    /// Debit
+    case debit
+
+    /// Credit
+    case credit
+}
+
+/// A Venmo line item to be displayed in the Venmo Paysheet.
+@objcMembers public class BTVenmoLineItem: NSObject {
+
+    // MARK: - Public Properties
+    
+    /// Number of units of the item purchased. This value must be a whole number and can't be negative or zero.
+    public var quantity: Int
+
+    /// Per-unit price of the item. Can include up to 2 decimal places. This value can't be negative or zero.
+    public var unitAmount: String
+
+    /// Item name. Maximum 127 characters.
+    public var name: String
+
+    /// Indicates whether the line item is a debit (sale) or credit (refund) to the customer.
+    public var kind: BTVenmoLineItemKind
+
+    /// Optional: Per-unit tax price of the item. Can include up to 2 decimal places. This value can't be negative or zero.
+    public var unitTaxAmount: String? = nil
+
+    /// Optional: Item description. Maximum 127 characters.
+    public var itemDescription: String? = nil
+
+    /// Optional: Product or UPC code for the item. Maximum 127 characters.
+    public var productCode: String? = nil
+
+    /// Optional: The URL to product information.
+    public var url: URL? = nil
+
+    // MARK: - Public Initializer
+    
+    /// Initialize a BTVenmoLineItem
+    /// - Parameters:
+    ///   - quantity: Number of units of the item purchased. Can include up to 4 decimal places. This value can't be negative or zero.
+    ///   - unitAmount: Per-unit price of the item. Can include up to 4 decimal places. This value can't be negative or zero.
+    ///   - name: Item name. Maximum 127 characters.
+    ///   - kind: Indicates whether the line item is a debit (sale) or credit (refund) to the customer.
+    @objc(initWithQuantity:unitAmount:name:kind:)
+    public init(quantity: Int, unitAmount: String, name: String, kind: BTVenmoLineItemKind) {
+        self.quantity = quantity
+        self.unitAmount = unitAmount
+        self.name = name
+        self.kind = kind
+    }
+    
+    // MARK: - Public Methods
+
+    /// Returns the line item in a dictionary.
+    /// - Returns: A dictionary with the line item information formatted for a request.
+    public func requestParameters() -> [String: Any] {
```

Comment:
Can this be internal? It looks like it's only used in this module.

Reply:
This is used here in the Venmo Client so I believe it cannot be internal - https://github.com/khushboo18/braintree_ios/blob/cb4058db8e9fad6c2be866088854fdab9dd512a2/Sources/BraintreeVenmo/BTVenmoClient.swift#L142

Reply:
Since the file it is accessed from also lives in the same module (BraintreeVenmo) we can use internal here! Internal is the default in Swift, so we can just remove the `public` keywork.